packages/bbui/src/Popover/Popover.svelte

```diff
@@ -18,6 +18,7 @@
   export let useAnchorWidth = false
   export let dismissible = true
```

Comment:
Do we not already have this `dismissible` prop to do exactly this - by setting it to false? Or does that do something else?

Reply:
Ooof, yeah. Good shout.